scripts/sli-report.sh

```
@@ -184,7 +184,7 @@ $REPORT
 BOD
 
 gh issue create --repo "$GOV_REPO" --title "SLI 周报 $WEEK（自动合并门禁自身指标）" \
-  --body-file "$TMP/body.md" --label sli-report || die "周报 issue 创建失败" \n  || gh issue create --repo "$GOV_REPO" --title "SLI 周报 $WEEK（自动合并门禁自身指标）" --body-file "$TMP/body.md"
+  --body-file "$TMP/body.md" || die "周报 issue 创建失败" \n  || gh issue create --repo "$GOV_REPO" --title "SLI 周报 $WEEK（自动合并门禁自身指标）" --body-file "$TMP/body.md"
```

Shellcheck (0.11.0)

[warning] \n is just literal 'n' here. For line feed, use a quoted, literal line feed instead.

(SC1012)